tests/settings.py

```diff
@@ -21,7 +22,7 @@
     },
     {
         "BACKEND": "django_rusty_templates.RustyTemplates",
-        "DIRS": ["tests/templates"],
+        "DIRS": ["tests/templates", Path(BASE_DIR) / "templates"],
```

Comment:
Can we move this into a test in [`tests/test_engine.py`](https://github.com/LilyFoote/django-rusty-templates/blob/eae574e935435f7e6c26352b45a71f0142f37812/tests/test_engine.py)?

Something like:

```python
def test_pathlib_dirs():
    engine = RustyTemplates({
        "OPTIONS": {},
        "DIRS": [Path(BASE_DIR) / "templates"],
    })
    template = engine.get_template("basic.txt")
    assert template.render({"user": "Lily"}) == "Hello Lily!"
```